xDSCResourceDesigner.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.2.0.0'
+ModuleVersion = '1.2.1.0'
```

Comment:
Set the version to 1.3.0.0 - this is inline with the convention we want to enforce.